docs/installation/installation.md

```diff
@@ -1,66 +1,72 @@
 ---
 id: install
 title: Install Netcode for GameObjects
-description: How to install Unity Netcode for GameObjects (Netcode). 
+description: How to install Unity Netcode for GameObjects (NGO). 
 ---
 
-Use this guide to install Unity Netcode for GameObjects (Netcode) 1.0.0 and later.
+Use this guide to install Unity Netcode for GameObjects (NGO) 1.0.0 and later.
 
 ## Prerequisites
 
 Before installing Netcode, you should ensure you have:
-  * An active Unity account with a valid license.
-  * A Netcode-supported installation of Unity. See [Netcode's requirements](#netcode-installation-requirements) for full details.
-  * An existing Unity project. If you do not have a project yet, we recommend using the [Hello World](../tutorials/helloworld.md) example to test the Netcode installation process and learn some of the basic features of Netcode.
+
+- An active Unity account with a valid license.
+- A Netcode-supported installation of Unity. See [Netcode's requirements](#netcode-installation-requirements) for full details.
+- An existing Unity project. If you don't have a project yet, see [Get started with NGO](../tutorials/get-started-with-ngo.md).
 
 ### Netcode Installation Requirements
 
 Netcode supports the following Unity versions:
-* Unity 2020.3, 2021.1, 2021.2, and 2021.3 LTS <!--Confirm versions-->
-* Mono and IL2CPP [Scripting Backends](https://docs.unity3d.com/Manual/scripting-backends.html)
+
+- Unity 2020.3, 2021.1, 2021.2, and 2021.3 LTS <!--Confirm versions-->
```

Comment:
Since the 2021.3 LTS is out, we can remove 2021.1 and 2021.2. I think we can just say 2020.3 and later, to be honest